proto/checksum/server_message.proto

```diff
@@ -25,9 +25,19 @@ option go_package = "github.com/temporalio/temporal/.gen/proto/checksum";
 
 import "event/server_message.proto";
 
+enum WorkflowExecutionState {
```

Comment:
this should probably go into an enum proto under temporal-proto/execution/, but that's a separate repo, so getting this reviewed first, to make sure the approach is solid.

Reply:
Per chat with @alexshtin, moving to [`proto/execution/server_enum.proto`](https://github.com/temporalio/temporal/pull/352/files#diff-18b7c6f09a838fa3afd5b65e3b3206b5).